Package.swift

```diff
@@ -3,9 +3,20 @@
 
 import PackageDescription
 
+// Excluding some Tools due to being tightly coupled to some unsupported frameworks on macOS
+#if os(macOS)
+let excludedSources: [String] = [
+    "Classes/CustomLocation",
+    "Classes/EmailSender"
+]
+#else
```

Comment:
I would rather check if it is `iOS` due to potential changes needed e.g. if we introduce `tvOS` support.